src/style.css

```diff
@@ -1 +1,86 @@
-/* Styles go here */
+@font-face {
+  font-family: Roboto;
+  src: url(https://fonts.googleapis.com);
+}
+
+@font-face {
+  font-family: Roboto;
+  font-weight: bold;
+  src: url(https://fonts.gstatic.com);
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  text-decoration: none;
+  list-style-type: none;
+  margin: 0;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  left: 50px;
+  top: 10px;
+  margin-left: 50px;
+}
+
+.nav__link {
+  position: relative;
+  display: inline-block;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  color: rgb(0, 0, 0);
+  margin: 20px;
+  padding-top: 22.5px;
+  padding-bottom: 22.5px;
```

Comment:
remove paddings, instead add a height 60px to the link, then center the text inside
also you don't need margins for all sides, just margin-right, also the margin-right for the last element in the list must be zero